### PR TITLE
Revert "bump canton to 2.8.0-snapshot.20230914.11170.0.v3bc03a0c (#17404)

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
Because artifacts have been overwritten.

### CODE REVIEW

**Review by raphael-speyer-da — APPROVED** [review]
LGTM to unblock but I'll leave it to others to know which is the correct version that should be used.

**Comment by remyhaemmerle-da** [comment]
included in #17416

**Pull request closed without merging**
